windows.sh

```diff
@@ -11,14 +11,13 @@
 
 ANSIBLE_PLAYBOOK=$1
 PLAYBOOK_DIR=${ANSIBLE_PLAYBOOK%/*}
-ROLE_REQUIREMENTS=$(find /vagrant/$PLAYBOOK_DIR -name "requirements.yml" -o -name "requirements.txt")
```

Comment:
This was explicitly added in #12 to make the role requirements location much more flexible. I'd rather not remove this feature...